molgenis-data-elasticsearch/src/main/java/org/molgenis/data/elasticsearch/index/MappingsBuilder.java

```diff
@@ -241,6 +240,15 @@ private static void createAttributeMappingContents(AttributeMetaData attr, boole
 				jsonBuilder.startObject("fields").startObject(FIELD_NOT_ANALYZED).field("type", "string")
 						.field("index", "not_analyzed").endObject().endObject();
 				break;
+			case SCRIPT:
+				jsonBuilder.field("type", "string");
+				jsonBuilder.field("norms").startObject().field("enabled", enableNorms).endObject();
+				// not-analyzed field for sorting and wildcard queries
+				// note: the include_in_all setting is ignored on any field that is defined in the fields options
+				// note: the norms settings defaults to false for not_analyzed fields
+				jsonBuilder.startObject("fields").startObject(FIELD_NOT_ANALYZED).field("type", "string")
+					.field("index", "no").endObject().endObject();
```

Comment:
This is a temporary code solution that must not be merged with the master. I suggest putting a // FIXME here and add it to the list of compute bugs to fix.